src/Neo/Wallets/Wallet.cs

```diff
@@ -661,6 +661,7 @@ public long CalculateNetworkFee(DataCache snapshot, Transaction tx)
                 }
             }
             networkFee += size * NativeContract.Policy.GetFeePerByte(snapshot);
+            networkFee += tx.GetAttributes<Conflicts>().Count() * tx.Signers.Count() * NativeContract.Policy.GetConflictsFee(snapshot);
```

Comment:
I prefer to define fee for any attribute, in the future could be better, please check https://github.com/neo-project/neo/pull/2916